### PR TITLE
Updated Commit comment event

### PR DESCRIPTION
**Pull request by smit-modi**
@svanharmelen Looking for a review

### CODE REVIEW

File: event_webhook_types.go [hunk 1]
```diff
@@ -112,18 +114,9 @@ type CommitCommentEvent struct {
...
-			AMode       string `json:"a_mode"`
-			BMode       string `json:"b_mode"`
-			NewFile     bool   `json:"new_file"`
-			RenamedFile bool   `json:"renamed_file"`
-			DeletedFile bool   `json:"deleted_file"`
-		} `json:"st_diff"`
-		Description string `json:"description"`
-		URL         string `json:"url"`
+		StDiff       *Diff  `json:"st_diff"`
+		Description  string `json:"description"`
```
Comment (svanharmelen) [inline event_webhook_types.go:118]: I don't see this field in the docs...

Comment (smit-modi) [reply event_webhook_types.go:118]: I haven't modified the Description field, it was already there.

Comment (smit-modi) [reply event_webhook_types.go:118]: I only changed `StDiff`'s type with `*Diff`

**Review by svanharmelen — APPROVED** [review]
Check... I'll take it 👍🏻

**Pull request merged**